### PR TITLE
Documentation explaining how to discover an S3 asset in OpenMetadata, and how to enrich it.

### PR DESCRIPTION
**Pull request by cdoron**
Signed-off-by: Doron Chen <cdoron@il.ibm.com>

### CODE REVIEW

**Comment by revit13** [comment]
@cdoron consider moving the page to tasks instead of sample. 
Also, as most of the details are not related to Fybrik, consider replacing them with specific links to openmetada data site so the burden of updating the instructions upon a change will be avoided.

**Comment by cdoron** [comment]
> @cdoron consider moving the page to tasks instead of sample. Also, as most of the details are not related to Fybrik, consider replacing them with specific links to openmetada data site so the burden of updating the instructions upon a change will be avoided.

Regarding your second comment, I think links to an OpenMetadata site would not be enough, because:

1. the creation of `s3` assets is not straightforward
2. some information here, such as information related to the custom properties, is Fybrik-specific

File: site/docs/samples/omd-discover-s3-asset.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
+# Discovering an S3 asset through the OpenMetadata UI
+
+This page explains how to discover an existing S3 asset through the OpenMetadata UI. The screenshots refer to `localstack` cloud storage, but the explanations also apply to other S3 services.
+
+Begin by opening your browser to the OpenMetadata UI. If you installed OpenMetadata in your kubernetes cluster in the `open-metadata` namespace, go to http://localhost:8585 after running:
+```
```
Comment (aradhalevy) [inline site/docs/samples/omd-discover-s3-asset.md:6]: in order to make http://localhost:8585 a link when running in the site,  wrap it in [ ] and add ( link )

File: site/docs/samples/omd-discover-s3-asset.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
+# Discovering an S3 asset through the OpenMetadata UI
+
+This page explains how to discover an existing S3 asset through the OpenMetadata UI. The screenshots refer to `localstack` cloud storage, but the explanations also apply to other S3 services.
```
Comment (aradhalevy) [inline site/docs/samples/omd-discover-s3-asset.md:3]: maybe a link to the read sample where the there is an option to make a S3 object can be helpful here.
something like : "existing S3 asset (to add a s3 assest refer to the read...) through the OpenMetadata UI"

**Review by aradhalevy — COMMENTED** [review]
I did the flow and it works great, also looks good in the site.

right now it is not in the site, to add this as a sample, add it to https://github.com/fybrik/fybrik/blob/master/site/mkdocs.yml under Notebook samples

**Review by aradhalevy — APPROVED** [review]

**Review by roytman — APPROVED** [review]

**Pull request merged**